simplecpp.cpp

```diff
@@ -1842,7 +1842,7 @@ namespace simplecpp {
                 return recursiveExpandToken(output, temp, loc, tok2, macros, expandedmacros2, parametertokens);
             }
 
-            else if (tok->str() == DEFINED) {
```

Comment:
please undo all changes in simplecpp.. these should be made in upstream and then they can be copied to cppcheck.

Reply:
We are currently in the simplecpp repo. 🙄

Reply:
oops!! :-)